src/mixins/widget.js

```diff
@@ -25,6 +25,11 @@ export const createWidgetMixin = ({ connector } = {}) => ({
 
       const { hydrated, started } = this.instantSearchInstance;
       if ((!started && hydrated) || this.$isServer) {
+        if (typeof this.instantSearchInstance.__forceRender !== 'function') {
+          throw new Error(
+            'You are using server side rendering with <ais-instant-search> instead of <ais-instant-search-server>.'
```

Comment:
The name of the widget is incorrect, it should be `ais-instant-search-ssr`.

https://github.com/algolia/vue-instantsearch/blob/20c2153983f45cf69834bde5f7ab474ac1db932b/src/components/InstantSearchSsr.js#L4

Reply:
fixed in 44f2d9c